src_assets/linux/misc/60-sunshine.rules

```diff
@@ -1,2 +1,14 @@
+# Allows Sunshine to acces /dev/uinput
 KERNEL=="uinput", SUBSYSTEM=="misc", OPTIONS+="static_node=uinput", TAG+="uaccess"
+
+# Allows Sunshine to access /dev/uhid
 KERNEL=="uhid", TAG+="uaccess"
+
+# Move virtual keyboard and mouse into a different seat
+SUBSYSTEMS=="input", ATTRS{id/vendor}=="ab00", ENV{ID_SEAT}="seat9", MODE="0660", TAG+="uaccess"
```

Comment:
With that change you will not be able to control the desktop. That's what we use in Wolf to avoid messing with the host session but it's definitely not something that you want in Sunshine 😉

I think you can safely remove the whole line since it's not needed to be "user accessable" it'll be the destkop environment to pick up those devices and access them without exposing them insecurely.